test/cppsuite/tests/bounded_cursor_stress.cpp

```diff
@@ -245,16 +243,18 @@ class cursor_bound_01 : public test {
      * key or with WT_NOTFOUND.
      */
     void
-    validate_bound_search(int range_ret, scoped_cursor &range_cursor, const std::string &search_key,
-      const bound &lower_bound, const bound &upper_bound)
+    validate_bound_search(int range_ret, scoped_cursor &bounded_cursor,
+      const std::string &search_key, const bound_set bounds)
```

Comment:
I think we can pass by reference `bounds` (same for `validate_bound_search_near`):
```suggestion
      const std::string &search_key, const bound_set &bounds)
```